examples/monitoring-alert/README.md

```diff
@@ -0,0 +1,28 @@
+## Requirements
+
+| Name | Version |
+|------|---------|
+| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 0.13 |
+| <a name="requirement_google"></a> [google](#requirement\_google) | ~> 4.0 |
+
+## Providers
+
+No providers.
```

Comment:
It should have a provider section.

See https://github.com/arthurlapertosa/terraform-google-secret-manager/blob/monitoring-destroy/examples/kms/README.md?plain=1#L7C3-L11C71